src/docs/chunkcache.dox

```diff
@@ -21,4 +21,10 @@ regardless of how infrequently they might be accessed.
 
 This ensures that the pinned tables are always relatively cheap to access, minimizing the performance overhead of accessing these tables.
 
+@section inserts Newly inserted or modified content
+
+We insert freshly-flushed data into the chunkcache to minimize latency and to reduce usage of resources in API access. Newer content is more likely to be accessed again so it is good to have the newer content available in the chunkcache. 
+This saves the chunkcache from having to read freshly-flushed data back from the object store soon after it was put there. When a tiered object is flushed and before it is deleted locally, we make an ingest call and the new or modified table will be inserted into the chunkcache. 
+This function will also check for outdated pinned content in the chunk cache, ensuring the older versions of the new chunks have their pinned flag removed so they may be cleaned up by chunk cache's eviction process. 
```

Comment:
I wouldn't mention the pinned flag specifically - it's an implementation detail. What about "... ensuring older versions of the pinned content can eventually be evicted"?